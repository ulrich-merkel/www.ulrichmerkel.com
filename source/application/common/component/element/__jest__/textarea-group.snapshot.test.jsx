/* eslint-disable import/no-extraneous-dependencies, func-names */
import React from 'react';
import renderer from 'react-test-renderer';
import ComponentToBeTested from './../textarea-group';

describe('component/element/textarea-group', function () {
    describe('Snapshot', function () {
        it('should render correctly', function () {
            const tree = renderer.create(
                <ComponentToBeTested
                    id='foo'
                    name='bar'
                    className='foobar'
                    label='test'
                    value='2'
                    isValid
                    isPristine
                >
                    Hello
                </ComponentToBeTested>
            ).toJSON();
            expect(tree).toMatchSnapshot();
        });
    });
});
