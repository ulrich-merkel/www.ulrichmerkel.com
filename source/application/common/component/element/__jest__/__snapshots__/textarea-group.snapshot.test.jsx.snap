exports[`component/element/textarea-group Snapshot should render correctly 1`] = `
<div
  className="foobar">
  <textarea
    aria-required={true}
    className="m-form__control--textarea is-pristine"
    cols="50"
    id="foo"
    name="bar"
    onBlur={[Function]}
    onChange={[Function]}
    required={true}
    rows="4"
    value="2" />
  <label
    className="m-form__label"
    htmlFor="foo">
    test
    <i
      aria-hidden="true"
      className="c-font-icon--sad m-form__label__icon--error" />
    <i
      aria-hidden="true"
      className="c-font-icon--smile m-form__label__icon--success" />
  </label>
</div>
`;
